src/components/Poster.tsx

```diff
@@ -0,0 +1,76 @@
+// src/components/Poster.tsx
+import React from 'react'
+import Image from 'next/image'
+import styles from './Poster.module.css'
+
+export type PosterProps = {
+  type: 'image' | 'gif' | 'video'
+  src: string
+  alt?: string            // for images/gifs
+  width?: number
+  height?: number
+  autoplay?: boolean      // only for video
+  loop?: boolean          // video + gif
+  muted?: boolean         // only for video
+}
+
+export const Poster: React.FC<PosterProps> = ({
+  type,
+  src,
+  alt = 'Poster',
+  width = 320,
+  height = 180,
```

Comment:
lets not hardcode the width and height. Poster should take as much space as it has. 

Think also how media (image/video) should be stretched when user uploaded horizontal image, and we display it on vertical screen. Maybe we should add some background to pad the media to the center